backend/simpleLambda/index.js

```diff
@@ -51,22 +51,23 @@ exports.handler = async (event, context) => {
                 newSentence = "";
             }
         }
+         
+        body = ('POST request success'   )
+
         // check if the array is empty
         if (sentenceArray.length == 0) {
-            sentenceObject.sentence = "No full sentences detected. Sentences must end with punctuation of either a period (.), exclamation point (!), or question mark (?), to be considered full sentences."
-            sentenceObject.type = "No full sentences detected to detect type. Sentences must end with punctuation of either a period (.), exclamation point (!), or question mark (?), to be considered full sentences.";
-            sentenceObject.sentiment = "No full sentences detected to determine sentiment. Sentences must end with punctuation of either a period (.), exclamation point (!), or question mark (?), to be considered full sentences.";
-            sentenceArray.push(sentenceObject);
+            statusCode = 400;
+            body = "No full sentences detected. Sentences must end with punctuation of either a period (.), exclamation point (!), or question mark (?), to be considered full sentences.";
         }
-        // return JSON array of sentences
-        body = sentenceArray;
         
     } catch (err) {
         statusCode = 400;
         body = err.message;
-    } finally {
-        body = JSON.stringify(body)
     }
     
-    return body;
+    return {
```

Comment:
Only the body should be returned, not the `statusCode` or `headers`

Reply:
i changed it. it only return the body.